content/en/examples/pods/config/redis-pod.yaml

```diff
@@ -15,10 +15,12 @@ spec:
       limits:
         cpu: "0.1"
     volumeMounts:
-    - mountPath: /redis-master-data
+    - mountPath: /data
       name: data
-    - mountPath: /redis-master
+    - mountPath: /usr/local/etc/redis
       name: config
+    args:
+    - /usr/loca/etc/redis/redis.conf
```

Comment:
Is `/usr/loca/etc/redis/redis.conf` a typo? (“loca”)

Reply:
Should not have been part of this commit. Removed.